code/modules/reagents/chemistry_reagents/medical.dm

```diff
@@ -581,6 +644,38 @@ datum/reagent/medicine/synaptizine/overdose_crit_process(mob/living/L, metabolis
 /datum/reagent/medicine/quickclot/overdose_crit_process(mob/living/L, metabolism)
 	L.apply_damages(0, 4*REM, 4*REM)
 
+
+/datum/reagent/medicine/quickclot_plus
+	name = "Quick Clot Plus"
+	description = "A chemical designed to completely repair internal bleeding. Significant toxic and shock side effects, do not use in combat."
+	color = "#CC00FF"
+	overdose_threshold = REAGENTS_OVERDOSE/30 //ONE unit.
+	overdose_crit_threshold = REAGENTS_OVERDOSE_CRITICAL/25 //Two units
```

Comment:
Use multiplications instead of divisions.

Reply:
same as before, weird repeating digit thing coming right up.

Reply:
changed